example/server/api/index.js

```diff
@@ -1,32 +1,22 @@
 const Router = require('express').Router;
-const environmentID = 'uCDQzKWgejrutqSYYsKWen';
-const flagsmith = require('../../../');
-const NodeCache = require('node-cache');
+const { Flagsmith } = require('../../../sdk');
```

Comment:
If I'm not mistaken this won't work since this is in typescript now, we'd have to first build the SDK and point this example to ../build ?

Reply:
I think also as a nice to have we adjust the example start script to 

```
    "start": "cd ../ && npm run build && cd ./example && node --inspect ./server",
```